test/Sentry.Testing/FakeTransportWithRecorder.cs

```diff
@@ -0,0 +1,34 @@
+namespace Sentry.Testing;
```

Comment:
i would kill the namespace here since it is redundant

Reply:
It's not redundant.  All the other files in this project have it, and removing it doesn't compile.